src/main/kotlin/com/faforever/userservice/ui/layout/OAuthLayout.kt

```diff
@@ -26,8 +26,8 @@ class OAuthLayout(oAuthHeader: OAuthHeader) : VerticalLayout(), RouterLayout {
     }
 
     init {
-        setHeightFull()
-        setWidthFull()
+        isPadding = false
+        isSpacing = false
```

Comment:
You can just have it extend CompactVerticalLayout instead of VerticalLayout

Reply:
didn't notice we already had custom base layouts for that.

Reply:
somehow it's not working with the CompactVerticalLayout, not a kotlin-dev, so would be nice if you could tell me what's going on here.
is the init of the parent not called, or set to the wrong layout?

Reply:
The no-arg constructor of the super class should be called whenever the child object is created.

Is the object still having the padding and spacing classes applied when you run it locally?

Reply:
yeah.. thought it was some local cache, but it seams to have some other origin.

Reply:
got a new hint, spacing is gone, but padding is applied. i'll debug and look where this comes from

Reply:
@Sheikah45 local issue, it works.
did somehow change the content of CompactVerticalLayout, but was not aware of it.